### PR TITLE
[ci] update doc for overlays

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: dev/ci/user-overlays/README.md [hunk 1]
```diff
@@ -5,27 +5,29 @@ have prepared a branch with the fix, you can add an "overlay" to your pull
...
+Example of an overlay for the project `elpi` that uses the branch `noinstance`
+from the fork of `SkySkimmer` and is active for pull request `13128`
 ```
 overlay elpi https://github.com/SkySkimmer/coq-elpi noinstance 13128
 ```
 
-See [`ci-common.sh`](../ci-common.sh) for the detailed documentation of
-the `overlay` function.
+Such a file can be created automatically using the scripts
+[`create_overlays.sh`]()../../dev/tools/create_overlays.sh).
```
Comment (Zimmi48) [inline dev/ci/user-overlays/README.md:28]:
```suggestion
[`create_overlays.sh`](../../dev/tools/create_overlays.sh).
```

**Review by Zimmi48 — APPROVED** [review]
Looks great! Just one typo.

**Comment by gares** [comment]
done

**Comment by Zimmi48** [comment]
@coqbot merge now

**Pull request merged**